modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalImportObjectDefinition.tsx

```diff
@@ -67,6 +67,15 @@ export default function ModalImportObjectDefinition({
 		},
 	});
 
+	const cleanData = () => {
+		setError(Liferay.Language.get('the-structure-failed-to-import'));
+		setExternalReferenceCode('');
+		setFile({
+			fileName: '',
+			inputFile: null,
+		});
```

Comment:
Hey @Artartur, very good solution! As this function will only be used in catch, you can put all of this inside catch, making the code easier to read:

```tsx
	catch (error) {
		setError(Liferay.Language.get('the-structure-failed-to-import'));
		setExternalReferenceCode('');
		setFile({
			fileName: '',
			inputFile: null,
		});
	}
```

Reply:
Done!